addons/stock_account/stock_account.py

```diff
@@ -24,6 +24,80 @@ def post_inventory(self, cr, uid, inv, context=None):
             ctx['force_period_date'] = inv.accounting_date
         return super(stock_inventory, self).post_inventory(cr, uid, inv, context=ctx)
 
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    @api.model
+    def _anglo_saxon_sale_move_lines(self, name, product, uom, qty, price_unit, currency=False, amount_currency=False, fiscal_position=False, account_analytic=False):
```

Comment:
add doc string, based on original function + explain new args
(especially types: are those IDs or records)

Use this format:

```python
"""Prepare dicts describing new journal COGS journal items for a product sale.

Returns a dict that should be passed to `_convert_prepared_anglosaxon_line()` to
obtain the creation value for the new journal items.

:param Model product: a `product.product` record of the product being sold
:param Model uom: a `product.uom` record of the UoM of the sale line
...
"""
```